.github/workflows/ci.yaml

```diff
@@ -120,7 +120,7 @@ jobs:
         include:
           - juju-snap-channel: "3.6/stable"
             agent-version: "3.6.0"
-            libjuju-version: "3.5.2.0"
+            libjuju-version: "3.6.0.0"
           - juju-snap-channel: "3.4/stable"
             agent-version: "3.4.3"
             libjuju-version:  "3.5.2.0"
```

Comment:
I updated the PR description to highlight the _destructive_ action rather than the _version refresh_ one, and implemented the changes in commit https://github.com/canonical/mysql-router-operator/pull/197/commits/436f2f7cb2bc536d83d4a0a2ed2ed27313ef4fd7.